pkgs/by-name/ka/kanboard/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nixosTests,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (attrs: {
+  pname = "kanboard";
+  version = "1.2.42";
+
+  src = fetchFromGitHub {
+    owner = "kanboard";
+    repo = "kanboard";
+    rev = "refs/tags/v${attrs.version}";
+    hash = "sha256-/Unxl9Vh9pEWjO89sSviGGPFzUwxdb1mbOfpTFTyRL0=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/kanboard
+    cp -rv . $out/share/kanboard
+
+    runHook postInstall
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = {
+      basic-frunctionality = nixosTests.kanboard;
+    };
+  };
+
+  meta = {
+    description = "Kanban project management software";
+    homepage = "https://kanboard.org";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ yzx9 ];
```

```suggestion
    maintainers = with lib.maintainers; [ yzx9 ];
    inherit (php.meta) platforms;
```
`mkDerivation` also doesn't set sensible platforms like some other builders, so we need to do it explicitly here